### DISCUSSION TITLE
Delete fix/28_prettier-pre-commit

### DISCUSSION THREAD

**Discussion by seporterfield** [Monstarrrr/rebutify#129] [General]
Ok to delete this branch?

Here are the changes that were in it:
https://github.com/Monstarrrr/rebutify/compare/main...fix/28_prettier-pre-commit

Take a good look and see if there's anything we're currently missing.
We can convert any missing changes into a new issue.
Then we delete the old branch.

**Comment by Monstarrrr** [comment]
i nuked it
